src/coreclr/tools/aot/ILCompiler.RyuJit/JitInterface/CorInfoImpl.Llvm.cs

```diff
@@ -66,7 +75,7 @@ public static void addCodeReloc(IntPtr thisHandle, void* handle)
             node.AppendMangledName(_this._compilation.NameMangler, sb);
             if (node is FrozenStringNode)
             {
-                sb.Append("___SYMBOL");
+                sb.Append("___SYMBOL\0");
```

Comment:
Do you need to append the `0` even when it is not `FrozenStringNode` ?

Reply:
Yes, sorry that was careless.  Moved after the `if` now.